packages/maskbook/src/social-network-adaptor/twitter.com/injection/PostDialogIcon.tsx

```diff
@@ -3,16 +3,12 @@ import { postEditorToolbarSelector } from '../utils/selector'
 import { createReactRootShadowed } from '../../../utils/shadow-root/renderInShadowRoot'
 import { PostDialogIcon } from '../../../components/InjectedComponents/PostDialogIcon'
 import { MaskMessage } from '../../../utils/messages'
-import { isCompose, isMobile } from '../utils/postBox'
 import { makeStyles, Theme } from '@material-ui/core'
 import { startWatch } from '../../../utils/watcher'
 
 export function injectPostDialogIconAtTwitter(signal: AbortSignal) {
     const emptyNode = document.createElement('div')
-    renderPostDialogIconTo(
-        postEditorToolbarSelector().map((x) => (isMobile() && isCompose() ? x : emptyNode)),
-        signal,
-    )
+    renderPostDialogIconTo(postEditorToolbarSelector(), signal)
```

Comment:
Please make sure no extra icons are injected because of the extending of the searching scope.

Reply:
It's a mistake. Thanks for pointing out.